src/common/logger.ts

```diff
@@ -34,10 +32,37 @@ const logLevel = process.env.LOG_LEVEL
       ? "error"
       : "info";
 
-const transports: Transport[] = [];
+interface Logger {
+  silly(...args: any[]): void;
+  warn(...args: any[]): void;
+  info(...args: any[]): void;
+  debug(...args: any[]): void;
+  error(...args: any[]): void;
+}
+
+function createLogger(): Logger {
+  if (ipcRenderer) {
+    return {
+      ...console,
+      // eslint-disable-next-line no-console
+      silly: (...args: any[]) => console.debug(...args),
+    };
+  }
 
-if (ipcMain) {
-  transports.push(
+  if (isTestEnv) {
```

Comment:
This feels a bit odd because it relies on a global state... would it be better to have it as a function arg?

Reply:
As a function argument to what? I could add a testing function that could be called to mock out the logger, is that what you mean?